code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -541,7 +548,14 @@
 		. += "[t_He] [t_is] wearing [head.get_examine_string(user)] on [t_his] head."
 	//suit/armor
 	if(wear_suit)
-		. += "[t_He] [t_is] wearing [wear_suit.get_examine_string(user)]."
+		//badge
+		var/badge_msg
```

Comment:
i really do not like the exclusiveness of it, make a component or generalize the vars cause other people may want to add attachables to suits as well